sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/AccessTier.java

```diff
@@ -55,7 +55,7 @@
     /**
      * Static value P60 for AccessTier.
      */
-    public static final AccessTier P60 = fromString("P60");
+    public static final AccessTier P65 = fromString("P60");
```

Comment:
This is just a validation that revapi work correctly right?

Reply:
Yup. It's With the "(REMOVE)" commit. Lol. Trying to see if it'll fail the storage build.